falco-no-driver.yaml

```diff
@@ -0,0 +1,111 @@
+package:
+  name: falco-no-driver
+  version: 0.36.1
+  epoch: 0
+  description: Cloud Native Runtime Security
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - falco-rules
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp
+      - abseil-cpp-dev
+      - abseillib
+      - autoconf
+      - automake
+      - bash
+      - binutils
+      - bpftool
+      - build-base
+      - busybox
+      - c-ares
+      - c-ares-dev
+      - ca-certificates-bundle
+      - clang-16
+      - clang-16-dev
+      - cmake
+      - curl-dev
+      - elfutils-dev
+      - gcc
+      - git
+      - grpc
+      - grpc-dev
+      - icu
+      - icu-dev
+      - jq-dev
+      - libbpf
+      - libbpf-dev
+      - libcurl-openssl4
+      - libelf
+      - libelf-static
+      - libsystemd
+      - libtbb-dev
+      - libtool
+      - libzstd1
+      - linux-headers
+      - llvm16
+      - m4
+      - make
+      - openssl
+      - openssl-dev
+      - patch
+      - perl
+      - protobuf
+      - protobuf-c-dev
+      - protobuf-dev
+      - re2
+      - re2-dev
+      - systemd-dev
+      - yaml-cpp
+      - yaml-cpp-dev
+      - yaml-dev
+      - zlib-dev
+      - zstd
+      - zstd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/falcosecurity/falco
+      tag: ${{package.version}}
+      expected-commit: 9eb611609a2876a5f5a5378e0613f0ff767f0d42
+      recurse-submodules: true
+
+  - runs: |
+      # Replace the find_dependency with find_package macro for newer cmake, otherwise it will fail.
+      sed -i 's/find_dependency(Protobuf CONFIG)/find_package(Protobuf CONFIG)/' /usr/lib64/cmake/grpc/gRPCConfig.cmake
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/falco
+      install -Dm755 ./falco.yaml "${{targets.destdir}}"/etc/falco/falco.yaml
+      sed -e 's/time_format_iso_8601: false/time_format_iso_8601: true/' < "${{targets.destdir}}"/etc/falco/falco.yaml
+
+  - working-directory: build
+    pipeline:
+      - runs: |
+          cmake \
+            -DCMAKE_INSTALL_PREFIX=/usr \
+            -DCMAKE_INSTALL_LIBDIR=/usr/lib \
+            -DFALCO_ETC_DIR=/etc/falco \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DUSE_BUNDLED_DEPS=Off \
+            -DMINIMAL_BUILD=On \
+            -DBUILD_DRIVER=On \
+            -DBUILD_LIBSCAP_MODERN_BPF=Off \
+            -DBUILD_BPF=Off \
```

Comment:
These flags look to be the same as `falco`. 

Aren't `-DBUILD_FALCO_MODERN_BPF=On` and `-DBUILD_DRIVER=Off` necessary?

Reply:
Yep, it seems these are exact flags that we already do in `falco`, since both flags are disabled by default, i guess.